src/data_scraper/data_scraper.py

```diff
@@ -1,11 +1,10 @@
+import credentials
 from src.data_scraper import time_helpers
 from src.api_client.api_client import BinanceClient
-from src import utils
-from src import config
-import credentials
+import src.utils as utils
+import src.config as config
```

Comment:
What's the difference between `from src import utils` and `import src.utils as utils`?